### PR TITLE
Changes according to ben's code

### PR DESCRIPTION
**Pull request by teojunjie**

### CODE REVIEW

File: src/data_loader.py [hunk 1]
```diff
@@ -89,24 +92,26 @@ def __iter__(self):
...
 
                 # target shape
                 target_shape = (720, 1280)
                 divide = 2**num_pool
                 gt_target_shape = (720//divide, 1280//divide)
 
                 # resizing with cv2
-                img_resized = cv2.resize(img, target_shape, interpolation = cv2.INTER_CUBIC)
-                gt_resized = cv2.resize(den, gt_target_shape, interpolation = cv2.INTER_CUBIC)
+                img_resized = cv2.resize(img, target_shape, interpolation = cv2.INTER_LINEAR)
```
Comment (benntay1) [inline src/data_loader.py:103]: for this one are u resizing?

File: src/data_loader.py [hunk 2]
```diff
@@ -29,7 +30,7 @@ def __init__(self, data_path, shuffle=False, pre_load=False, num_pool=None):
         if shuffle: random.seed(2468)
 
         self.data_files = [os.path.join(data_path, filename) for filename in os.listdir(data_path)
-                           if os.path.isfile(os.path.join(data_path, filename))][0:100]
+                           if os.path.isfile(os.path.join(data_path, filename))][0:300]
```
Comment (benntay1) [inline src/data_loader.py:33]: so now we working with the first 300 files?

Comment (teojunjie) [reply src/data_loader.py:33]: For smc yea... coz its taking too long

Comment (teojunjie) [reply src/data_loader.py:33]: I can change it back to 500 for the report but for the poster we see if we have sometime this week, if not its fine

File: src/data_loader.py (hunk 1 above)
Comment (teojunjie) [reply src/data_loader.py:103]: Oh am i not supposed to?

**Review by teojunjie — COMMENTED** [review]
Replied @benntay1

File: src/data_loader.py (hunk 1 above)
Comment (benntay1) [reply src/data_loader.py:103]: yeah no need to resize anymore

the resizing should be in crowd_count.py

File: src/data_loader.py (hunk 2 above)
Comment (benntay1) [reply src/data_loader.py:33]: ok!

**Review by benntay1 — APPROVED** [review]

**Pull request merged**